Source/HyprMXAdapter.swift

```diff
@@ -67,13 +64,16 @@ final class HyprMXAdapter: PartnerAdapter {
 
         // HyprMX.initialize() uses WKWebView, which must only be used on the main thread
         DispatchQueue.main.async { [self] in
-            // consentStatus will be updated by setGDPR & setCCPA after init
-            HyprMX.initialize(withDistributorId: distributorId,
-              userId: gameID,
```

Comment:
If gameID is no longer used, let's remove the lines above that fetch it.
Is there some new HyprMX API to set the gameID, maybe independent from the initialize() method, that we should be using instead?

Reply:
Nvm, I checked their changelog and it seems they don't require this anymore.

Reply:
Oh, looks like I need to delete some code still.

Reply:
Done.